ux.symfony.com/config/packages/symfonycasts_sass.yaml

```diff
@@ -0,0 +1,2 @@
+symfonycasts_sass:
+    #binary: '/opt/homebrew/bin/sass'
```

Comment:
I think this isn't needed

Reply:
Yeah some things are pished "as it was"... sorry ;)